### PR TITLE
Matched func_802419F0_ECB6F0 in ECAA80.c

### PR DESCRIPTION
**Pull request by cfillak**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.00%) on `us`.

File: src/ECAA80.c [hunk 1]
```diff
@@ -25,7 +25,16 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
 INCLUDE_ASM(s32, "ECAA80", func_8024199C_ECB69C);
 
 // ECAA80_ItemChoice_SaveSelected (needs data?)
```
Comment (nanaian) [inline src/ECAA80.c:27]: Can you rename `func_802419F0_ECB6F0` to `ECAA80_ItemChoice_SaveSelected`? Should be a find+replace

File: src/ECAA80.c [hunk 2]
```diff
@@ -25,7 +25,16 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
...
-INCLUDE_ASM(s32, "ECAA80", func_802419F0_ECB6F0);
+
+extern s32 ECAA80_ItemChoice_HasSelectedItem;
+extern s32 ECAA80_ItemChoice_SelectedItemID;
+
+s32 func_802419F0_ECB6F0(Evt *arg0) {
```
Comment (nanaian) [inline src/ECAA80.c:32]:
```suggestion
ApiStatus ECAA80_ItemChoice_SaveSelected(Evt* script, s32 isInitialCall) {
```

File: src/ECAA80.c [hunk 3]
```diff
@@ -25,7 +25,16 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
...
+    ECAA80_ItemChoice_SelectedItemID = dead_evt_get_variable(arg0, *arg0->ptrReadPos);
+    ECAA80_ItemChoice_HasSelectedItem = 1;
+    return 2;
```
Comment (nanaian) [inline src/ECAA80.c:35]:
```suggestion
    return ApiStatus_DONE2;
```

File: src/ECAA80.c [hunk 4]
```diff
@@ -25,7 +25,16 @@ void ECAA80_UnkFloAI_Chase(Evt* script, MobileAISettings* aiSettings, EnemyDetec
 INCLUDE_ASM(s32, "ECAA80", func_8024199C_ECB69C);
 
 // ECAA80_ItemChoice_SaveSelected (needs data?)
-INCLUDE_ASM(s32, "ECAA80", func_802419F0_ECB6F0);
+
+extern s32 ECAA80_ItemChoice_HasSelectedItem;
+extern s32 ECAA80_ItemChoice_SelectedItemID;
+
+s32 func_802419F0_ECB6F0(Evt *arg0) {
+    ECAA80_ItemChoice_SelectedItemID = dead_evt_get_variable(arg0, *arg0->ptrReadPos);
```
Comment (nanaian) [inline src/ECAA80.c:33]:
```suggestion
    ECAA80_ItemChoice_SelectedItemID = dead_evt_get_variable(arg0, *arg0->ptrReadPos++);
```

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/ECAA80.c (hunk 4 above)
Comment (ethteck) [reply src/ECAA80.c:33]: you need to use an args temp and increment that instead (search for other functions that return ApiStatus_DONE2) - this applies for all funcs in this PR

File: src/EE70C0.c [hunk 5]
```diff
@@ -16,7 +16,15 @@ extern const s32 D_802453E0_EEB990[];
...
 
-INCLUDE_ASM(s32, "EE70C0", func_802422B4_EE8864);
+extern s32 D_8024462C_EEABDC;
+extern s32 D_80244630_EEABE0;
+
+s32 func_802422B4_EE8864(Evt *arg0) {
+    D_80244630_EEABE0 = dead_evt_get_variable(arg0, *arg0->ptrReadPos);
+    D_8024462C_EEABDC = 1;
+    return 2;
+}
```
[the remark this answers is not in the document]
Comment (cfillak) [reply src/EE70C0.c]: Should I also rename this one from "func_802422B4_EE8864" to "EE70C0_ItemChoice_SaveSelected" in keeping with the renaming of the other function in this PR?

File: src/ECAA80.c (hunk 4 above)
Comment (cfillak) [reply src/ECAA80.c]: Ok, makes sense. Those changes are now implemented.

**Review by nanaian — CHANGES_REQUESTED** [review]
LGTM besides double empty line before each function

File: src/EE70C0.c (hunk 5 above)
[the remark this answers is not in the document]
Comment (cfillak) [reply src/EE70C0.c]: Alright, just changed it with find and replace.

**Review by nanaian — APPROVED** [review]

**Pull request merged**